packages/app/src/components/FlameColorEditor/FlameColorEditor.module.css

```diff
@@ -14,6 +14,7 @@
 }
 
 .canvas {
+  z-index: -1;
```

Comment:
For ios, svg components were not shown because canvas was blocking them

Reply:
Huh, really not sure what is the issue here, also something to check later, I currently don't have time to go too deep into every little shenanigans for ios implementation, but this seems to work on every device now, if you have any good tricks on how to check this stacking and what does ios do differently, let me know, but I figured it would take a lot of specs reading, to understand differences between impls.